src/models/generators/ParticleCluster.cpp

```diff
@@ -13,4 +13,11 @@ ParticleCluster::ParticleCluster(
     , meanVelocity(meanVelocity)
     , dimensions(dimensions)
 {
+    if (dimensions != 2 && dimensions != 3) {
+        spdlog::warn(
+            "The dimensions specified for the brownian motion of a particle cluster was set to {}. "
+            "It can only be of values 2 or 3",
+            dimensions);
+        throw std::invalid_argument("Dimensions must be 2 or 3.");
+    }
```

Comment:
Ich finde hier könnte man alle `dimensions` zwischen 0 und 3 zulassen, z.B. wenn man einfach mal die Brownian motion "ausschalten" will. Was würdest du sagen? Also dann:
```suggestion
    if (dimensions > 3) {
        spdlog::warn(
            "The dimensions specified for the brownian motion of a particle cluster was set to {}. "
            "It can only be of values smaller equal three",
            dimensions);
        throw std::invalid_argument("Dimensions must be smaller equal 3.");
    }
```